clients/include/utility.h

```diff
@@ -129,6 +140,33 @@ inline hipblasBfloat16 float_to_bfloat16(float f)
     return rv;
 }
 
+/* =============================================================================================== */
+/* Complex / real helpers.                                                                         */
+template <typename T>
```

Comment:
CI appears to be complaining about just about all of the templated functions from line 145 down. It may be related to that extra semicolon in the cblas_interface.cpp